sandbox.html

```diff
@@ -159,6 +160,10 @@
     <label for="embeddingTargetId">Embedding target ID *</label>
     <input name="embeddingTargetId" id="embeddingTargetId" required>
   </div>
+  <div>
+    <label for="mediaType">MediaType (optional, for Jasper Reports)</label>
+    <input name="mediaType" id="mediaType" placeholder="application/pdf">
```

Comment:
Imo it doesn't make sense to use a specific value as the placeholder. In this case (since only PDF supported), I'd use a checkbox "Show Jasper Report as PDF".

Reply:
Sounds good, I'll change it.